app/views/layouts/application.html.erb

```diff
@@ -8,7 +8,30 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
-  <body>
-    <%= yield %>
-  </body>
+  <nav class="navbar navbar-inverse">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <%= link_to 'Flash Site', root_url, class: "navbar-brand" %>
+      </div>
+      <div class="nav navbar-nav navbar-right">
+        <% if current_user %>
+           <strong id= "user_name"><%= current_user.first_name.capitalize %></strong> <%= link_to "Logout", logout_path%>
```

Comment:
@jynti Please make sure there are no indentation issues in the code.

Reply:
Done.